integrations/dojo.md

```diff
@@ -8,12 +8,12 @@ keywords: integration integrate dojo
 
 ## TinyMCE 4 Widget for Dojo
```

Comment:
Should we remove the `4` from the title and para below? Looking at the code, it looks pretty generic so I'd expect it'd work for v5 as well.

Reply:
I want to test it a little first. that page will be re-written soon (possibly by the end of the year)